presto-main/src/main/java/io/prestosql/operator/HttpPageBufferClient.java

```diff
@@ -387,26 +387,26 @@ public Void handle(Request request, Response response)
                             }
                         });
                     }
+
```

Comment:
Could we have a test in `io.prestosql.operator.TestHttpPageBufferClient` for this case (e.g the exception did not bubble up)?

Reply:
Test added